lola.html

```diff
@@ -806,11 +835,12 @@ <h4>Other Activity Types</h4>
         <b>Question</b> activities SHOULD be copied but the destination MAY close them when copying if they are not closed.
       </p>
 
-      <h3>Copying or Setting Permissions</h3>
+      <h3>Inferring and Setting Permissions</h3>
 
-      <p>
-        Although a follower list may not be the same on old and new servers, it seems like it would fit most users’ intended use cases if content addressed to their followers on their old server is now readable by their followers on a new server.  Otherwise, a destination may allow the user to select appropriate permissions before or after an account migration through their own UIs.
-      </p>
+      <p>It would be really great to have interoperable schemas for permissions.  In the meantime, even without explicit permissions, a destination can make some pretty good choices.</p> 
```

Comment:
there's not a FEP for it, but the "Authorized Fetch" thing mentioned in #23 is... maybe worth discussing on a call at least (assuming there are enough people on the call that know how it works better than I do!)?